main/User_config.h

```diff
@@ -285,6 +296,6 @@ uint8_t wifiProtocol = 0; // default mode, automatic selection
 #define subjectMQTTtoSYSset          "/commands/MQTTtoSYS/config"
 
 /*-------------------DEFINE LOG LEVEL----------------------*/
-#define LOG_LEVEL LOG_LEVEL_NOTICE
+#define LOG_LEVEL LOG_LEVEL_TRACE
```

Comment:
Hello Jon,
Could you put it back to NOTICE level please

Reply:
Oops. Re-pushing now